components/director/internal/model/labeldef.go

```diff
@@ -0,0 +1,34 @@
+package model
+
+import (
+	"github.com/kyma-incubator/compass/components/director/pkg/jsonschema"
+	"github.com/pkg/errors"
+)
+
+type LabelDefinition struct {
+	ID     string
+	Tenant string
+	Key    string
+	Schema *interface{}
+}
+
+func (def *LabelDefinition) Validate() error {
+	if def.ID == "" {
```

Comment:
Do we really need to validate these fields? Schema should be validated, but what about other values? The non-empty values are enforced in DB and GraphQL API, so it's rather not necessary.

Reply:
in GraphQL we mark that some fields are required, but still, you can provide empty string there, which is not proper value. If object is not valid, there is not too much sense to call DB at all.